docs/docsite/rst/dev_guide/testing/sanity/no-wildcard-import.rst

```diff
@@ -1,29 +0,0 @@
-Sanity Tests » no-wildcard-import
```

Comment:
We might want to leave the docs in place, but simply note that the test has been superseded by pylint.

Otherwise old links on PR test failures will end up linking to a non-existent page.